rust/build.rs

```diff
@@ -59,8 +62,10 @@ fn make_librunecoral() -> PathBuf {
     let user_id = id("-u");
     let group_id = id("-g");
 
-    // FIXME: Parameterize the target CPU architecture
-    let cpu = "k8";
+    let cpu = match std::env::var("CARGO_CFG_TARGET_ARCH").unwrap().as_ref() {
+        "x86_64" => "k8".to_string(),
```

Comment:
Won't this fail when `CARGO_CFG_TARGET_OS` is `windows` or `darwin`?

Reply:
It will fail. For now our makefile doesn't even support building on windows.

```Make
ifeq ($(CPU),k8)
RUNE_CORAL_DIST_DIR := $(MAKEFILE_DIR)/dist/lib/linux/x86_64
else ifeq ($(CPU),aarch64)
BAZEL_BUILD_FLAGS_Linux += --copt=-ffp-contract=off
RUNE_CORAL_DIST_DIR := $(MAKEFILE_DIR)/dist/lib/linux/aarch64
else ifeq ($(CPU),armv7a)
BAZEL_BUILD_FLAGS_Linux += --copt=-ffp-contract=off
RUNE_CORAL_DIST_DIR := $(MAKEFILE_DIR)/dist/lib/linux/armv7l
else ifeq ($(CPU), darwin)
RUNE_CORAL_DIST_DIR := $(MAKEFILE_DIR)/dist/lib/darwin
endif
```

So the idea is to slowly add support for each platform by adding more 
```
build_librunecoral_windows()
build_librunecoral_darwin()
```
functions in the build.rs .

Should i rename this function to build_librunecoral_linux() now?

Reply:
Na, let's leave it as is and then when support is added for more plaforms we'll update it.